pkg/kubernetes/views/imagestream-body.html

```diff
@@ -1,3 +1,16 @@
+<div ng-repeat="statustags in stream.status.tags">
+  <div ng-repeat="condition in statustags.conditions" ng-if='condition.status == "False"'
```

Comment:
I think we should target this to ImportSuccess conditions only, is that is the only implemented one right now. ei: ng-if="condition.type == 'ImportSuccess' && condition.status == "False"'. Makes the rest of this simpler, we we can remove the condition.reason and condition.type variables from the status string below, and remove the $first check from the edit image stream link